dart/test/server_test.dart

```diff
@@ -254,6 +255,63 @@ void main() {
             'details': "The stream '$streamId' is not subscribed",
           }))));
     });
+
+    group('_Service', () {
+      final serviceStream = '_Service';
+
+      test('gives register and unregister events', () async {
```

Comment:
Let's add a test for when a client registers with previously disconnected clients. It should not receive corresponding disconnected events.

Reply:
done